### ISSUE TITLE
Css import error

### ISSUE THREAD

**Issue by lfirek** [airbnb/react-dates#764]
Hello,

I finded bug in actual npm package, version ``13.0.2``.

``import 'react-dates/lib/css/_datepicker.css';`` 
gives error because in ``_datepicker.css`` one of classes had caption with : [object Object]
```
.CalendarMonth_table {
  border-collapse: collapse;
  border-spacing: 0;
  caption: [object Object]
}
```

**Comment by majapw** [comment]
I have fixed this... but haven't done a new release! Will do that presently. :)

**Comment by majapw** [comment]
Should be fixed in 13.0.3!

**Comment by lfirek** [comment]
@majapw Dzięki Maja :smile: :poland:
